app/services/rapid_ready_for_decision/hypertension_upload_manager.rb

```diff
@@ -4,15 +4,25 @@ module RapidReadyForDecision
   class HypertensionUploadManager
     attr_accessor :submission
 
+    DOCUMENT_TITLE = 'VAMC_Hypertension_Rapid_Decision_Evidence'
+
     def initialize(submission)
       @submission = submission
     end
 
+    def file_upload_name
```

Comment:
It looks like you're calling `file_upload_name` multiple times. There's a minuscule chance that you'll get a different value if the job is running at midnight when the date changes. A solution is to memoize the return value.

Reply:
Thanks!

Reply:
That is one reason why I had `date` as an instance variable.